### DISCUSSION TITLE
URL Filtering Without Page Refreshes

### DISCUSSION THREAD

**Discussion by JoshKornfeld** [SafeExamBrowser/seb-win-refactoring#884] [Q&A] [answered]
How do you block URLs when a page is not refreshed? For example, if I enable the URL https://vercel.com/docs, then https://vercel.com/pricing is accessible despite it not being in the URL filters. 

Thanks :)

**Comment by danschlet** [comment, marked as answer]
You need to activate the option "Filter also embedded content" in the [Network/Filter](https://safeexambrowser.org/windows/win_usermanual_en.html#NetworkPaneFilterSection) tab.
